requirements.txt

```diff
@@ -1,12 +1,12 @@
-black==22.1.0
+black==22.3.0
 faiss-cpu==1.7.2
 flake8==4.0.1
 lmdb==1.3.0
-onnxruntime==1.10.0
-pytorch-lightning==1.5.10
-regex==2022.3.2
+onnxruntime==1.11.1
+pytorch-lightning==1.6.4
+regex==2022.6.2
 sentencepiece==0.1.96
-streamlit==1.6.0
-torch==1.10.2
-tqdm==4.63.0
-transformers[ja]==4.16.2
+streamlit==1.10.0
+#torch==1.11.0  # You may want to install torch outside this file for the CUDA compatibility.
```

Comment:
Since there are different install options for PyTorch depending on environment, we recommend the users to install PyTorch outside the requirements.txt.